packages/preview-middleware-client/src/flp/WorkspaceConnector.ts

```diff
@@ -79,6 +80,10 @@ const connector = merge({}, ObjectStorageConnector, {
             features.isVariantAdaptationEnabled = false;
         }
 
+        if(settings?.scenario === 'ADAPTATION_PROJECT') {
```

Comment:
Please add also a space between the `if` and the `(` in order to keep the formatting.

Reply:
Done